templates/spark/v1/spark.jinja

```diff
@@ -0,0 +1,112 @@
+{% set WORKERS = properties['workers'] if properties and properties['workers'] else 2 %}
```

Comment:
A good alternative here someone used is to add a variable {% set PROPERTIES = properties or {} %}, then all of your other checks can just be "= properties['foo'] or X"